experimental/generation/generator/src/commands/dialog/generate.ts

```diff
@@ -28,7 +28,8 @@ export default class GenerateDialog extends Command {
         schema: flags.string({ char: 's', description: 'Path to your app.schema file.', required: false }),
         singleton: flags.boolean({ description: 'Specify to generate a single .dialog file.' }),
         templates: flags.string({ char: 't', description: 'Directory with templates to use for generating assets.  With multiple directories, the first definition found wins.  To include the standard templates, just use "standard" as a template directory name.', multiple: true }),
-        verbose: flags.boolean({ description: 'Output verbose logging of files as they are processed', default: false })
+        verbose: flags.boolean({ description: 'Output verbose logging of files as they are processed', default: false }),
+        jsonProperties: flags.string({ char: 'j', description: 'The additional json properties to be added into the scope', required: false})
     }
```

Comment:
These are sorted alphabetically, but I don't understand the purpose of this flag.